site/src/App/Code/Code.treat.ts

```diff
@@ -0,0 +1,48 @@
+import { style } from 'sku/treat';
+
+export const button = style({
+  cursor: 'pointer',
+  outline: 'none',
+});
+
+const overlay = style({
+  transition: 'opacity 0.2s',
```

Comment:
`FieldOverlay` already sets `transition="fast"` so this should be redundant.

Reply:
I copied this from our button code, so this redundancy is duplicated there too.

Reply:
Cleaned this up in a separate PR: https://github.com/seek-oss/braid-design-system/pull/267